packages/react-components/src/components/Search/Search.module.scss

```diff
@@ -0,0 +1,90 @@
+$base-class: 'search';
```

Comment:
Have we abandoned 'lc-something' convention?

Reply:
https://github.com/livechat/design-system/pull/291#discussion_r845897391

Reply:
However, I would re-think the naming of the class and the component itself. `Search` is too generic for me. I would suggest `SearchInput`. 

What do you think guys?

Reply:
☝️